### PR TITLE
Fix Bug to display same status information in UserList and UserDetail Page

### PR DESCRIPTION
**Pull request by RamuKammagani-osi**

### CODE REVIEW

File: app/javascript/selectors/detailSelector.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ export const selectDetailRecords = state => {
   const userDetails = usersObject ? usersObject.records : {};
   userDetails.enabled === undefined
     ? (userDetails.enabled = '')
-    : userDetails.enabled
+    : userDetails.enabled === true
```
Comment (kelfink) [inline app/javascript/selectors/detailSelector.js]: Seems strange.  Javascript weirdness ?

File: app/javascript/selectors/detailSelector.js [hunk 2]
```diff
@@ -1,11 +1,9 @@
 export const selectDetailRecords = state => {
   const usersObject = state.fetchDetails ? state.fetchDetails.details : null;
   const userDetails = usersObject ? usersObject.records : {};
-  userDetails.enabled === undefined
-    ? (userDetails.enabled = '')
-    : userDetails.enabled
-      ? (userDetails.enabled = 'Active')
-      : (userDetails.enabled = 'Inactive');
+  userDetails.enabled === true || userDetails.enabled === 'Active'
```
Comment (kelfink) [inline app/javascript/selectors/detailSelector.js]: good! simpler!

File: app/javascript/views/userDetail/UserDetail.jsx [hunk 3]
```diff
@@ -56,6 +56,38 @@ export default class UserDetail extends Component {
     }
   };
 
+  renderCards = permissionsList => {
```
Comment (kelfink) [inline app/javascript/views/userDetail/UserDetail.jsx]: making code climate happy!  Some day we should put this in its own file and make it a component.  Maybe

File: app/javascript/views/userDetail/UserDetail.jsx [hunk 4]
```diff
@@ -91,25 +123,7 @@ export default class UserDetail extends Component {
...
-              enableSave={this.state.enableSave}
-              permissionsList={permissionsList}
-            />
-          ) : (
-            <UserDetailShow
-              details={this.state.details}
-              onEdit={() => this.setState({ isEdit: true })}
-            />
-          )}
+          {this.renderCards(permissionsList)}
```
Comment (kelfink) [inline app/javascript/views/userDetail/UserDetail.jsx]: Feels weird to pass permissionsList along, but it's what we're doing and it works.

File: app/javascript/views/userDetail/UserDetail.jsx (hunk 3 above)
Comment (RamuKammagani-osi) [reply app/javascript/views/userDetail/UserDetail.jsx]: Yeah! We have to when the component grows..!

**Pull request merged**